README.md

```diff
@@ -72,12 +72,12 @@ fun MyScreen() {
 
 ## Structure
 
-- **`audiowaveformview/`**: Core TimelineView library module
+- **`timelineview/`**: Core TimelineView library module
 - **`app/`**: Demo application showcasing various use cases
 
 ## Documentation
 
-For detailed API documentation, see [audiowaveformview/README.md](audiowaveformview/README.md)
+For detailed API documentation, see [timelineview/README.md](timelineview/README.md)
```

Comment:
https://github.com/ferPrieto/TimelineView/pull/32/files#diff-58e9104eb9546385330bf0e1217d3a53439f800907d613e25fcca70a24dafe9d broke this README's link

Reply:
No other important mentions of `audiowaveformview`

<img width="1093" height="673" alt="Screenshot 2025-07-30 at 3 11 25 PM" src="https://github.com/user-attachments/assets/40ab45cb-a741-44eb-87fe-b1562e7fe3e8" />

Reply:
Good catch!